### DISCUSSION TITLE
Request for AArch64 Support for DeepMD

### DISCUSSION THREAD

**Discussion by siinghim** [deepmodeling/deepmd-kit#4453] [Q&A]
I am currently trying to use DeepMD on an AArch64 architecture, but I noticed that the Conda package for DeepMD does not currently support AArch64. This is causing difficulties in installing and using the software. Could you consider adding AArch64 support for DeepMD in future releases or provide guidance on how to build it from source on AArch64 systems?

**Comment by njzjz** [comment]
This was discussed in https://github.com/deepmodeling/deepmd-kit/issues/3932

**Reply by siinghim** [reply]
Thanks！
